### DISCUSSION TITLE
Tasks

### DISCUSSION THREAD

**Discussion by HafnerMichael** [AIS-22/UNI-AIS-MediaDataFormats#1] [General]
![exercises](https://user-images.githubusercontent.com/116754128/226181075-b77995a1-3d20-4f11-9028-d74722400352.JPG)

**Comment by HafnerMichael** [comment]
Hab mal nach Beschreibungen und Code für unser KI Thema gesucht und auf meinen Branch gepusht

**Comment by m-langer** [comment]
What we need:

- Fixed File size
  - change quality for each image until a certain file size is reached (JPEG), 
  - approach the file size bound from below
- how to determine; how to classify 
  - deep learning based approach
  - all other approaches (feature extraction, classification)
    - extraction:
      - Histogram
      - local line
    - classify:
      - svm
      - k-means

**Reply by m-langer** [reply]
Classifier:
- extraction histogramme
- deep learning approach

compr3ession
- JPEG XL
- all other in a) b)

next steps:
- apply compression
- check if file size is really fixed
